shapes/README.md

```diff
@@ -48,11 +48,14 @@ It should be >= 7.0
     Use constructor inheritance to allow initialization of all shapes providing only a `Color` as a parameter. Create some shapes providing `Color` only param.
     Add in-class field initialization for all shapes to safely use inherited constructor.
 9.  SFINAE
-    Write a function that allows inserting only subclasses of Shape to the collection. Other parameter types should not compile. Use SFINAE. Find proper type_traits.
+    Write a function that allows inserting only subclasses of Shape to the collection.
+    Other parameter types should not compile.
+    Use SFINAE.
+    Find proper type_traits.
```

Comment:
@ziobron What should this function do? What should be its name, arguments etc.?

Reply:
Something like that:

```cpp
template <typename Sh, typename Coll, typename = std::enable_if_t<std::is_base_of_v<Shape, Sh>>
void addShape(Shape sh, Coll coll) {
    coll.emplace_back(sh);
}
```

(written from hand, check if it compiles)